install/kubernetes/helm/istio/charts/pilot/values.yaml

```diff
@@ -12,8 +12,10 @@ rollingMaxUnavailable: 25%
 image: pilot
 sidecar: true
 traceSampling: 1.0
-# if protocol sniffing is enabled. Default to true.
-enableProtocolSniffing: true
+# if protocol sniffing is enabled for outbound. Default to true.
```

Comment:
nit - Just remove the `default to...`. If they are reading this they can see the default (and its not accurate :slightly_smiling_face: )